src/pages/AccessPage.tsx

```diff
@@ -139,8 +140,10 @@ const AccessPage = () => {
         <div className="content">
           <TextileLogo />
           <Switch>
-            <Route path="/signup" component={SignUpForm} />
-            <PrivateRoute path="/" component={SuccessForm} />
+            <Route path="/success" component={SuccessForm} />
```

Comment:
we're not using private routes for now, and we'll redirect everything except success to the base url for now